CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ option(NCNN_PIXEL_AFFINE "warp affine image pixel" ON)
 option(NCNN_PIXEL_DRAWING "draw basic figure and text" ON)
 option(NCNN_CMAKE_VERBOSE "print verbose cmake messages" OFF)
 option(NCNN_VULKAN "vulkan compute support" OFF)
+option(NCNN_WEBGPU "webgpu compute support" OFF)
```

Comment:
Do not provide another webgpu option
When compiling to wasm target with emscripten, enable webgpu according to `NCNN_VULKAN` option